perl6.html.markdown

```diff
@@ -4,6 +4,7 @@ language: perl6
 filename: learnperl6.pl
 contributors:
     - ["vendethiel", "http://github.com/vendethiel"]
+    - ["samcv", "https://gitlab.com/u/samcv/"]
```

Comment:
I'm sorry, but we reserve the contributor tag for more major contributions. LGTM otherwise!